core/src/main/java/com/github/mizool/core/rest/errorhandling/GenericErrorMapper.java

```diff
@@ -65,13 +70,16 @@ private void logError(Throwable t, ErrorHandlingBehavior behavior)
         Throwable rootCause = determineRootCause(t);
         if (rootCause != t)
         {
-            behavior.getMessageLogLevel().log(log, "{} - {}", t.getMessage(), rootCause.getMessage());
+            behavior.getMessageLogLevel()
+                .log(log, "{} - {}", t.getMessage(), rootCause.getMessage());
         }
         else
         {
-            behavior.getMessageLogLevel().log(log, t.getMessage());
+            behavior.getMessageLogLevel()
```

Comment:
Where do all these weird short line breaks come from?

Reply:
In the spirit of "each operation should have it's own line", `Chained method calls: Wrap always` was enabled in the code style settings on [August 4, 2020](https://github.com/mizool/intellij-settings/commit/d4592b36261d415b3ab6eb3b00208f2f5110406c).

I think this was mainly intended for chained calls when working with builders or streams. Sadly the IntelliJ code formatter does not distinguish between different types of call chains.